docs/definition_v1beta1/services.rst

```diff
@@ -0,0 +1,2 @@
+Services for Google Cloud Aiplatform V1beta1 Schema Trainingjob Definition v1beta1 API
```

Comment:
Is the generator producing this file? It seems to be missing the module.

You may need to ignore some of the files (by adding to `excludes` in `synth.py`)

Reply:
Yeah the generator produced that file. I'll remove it from this PR.